scripts/install.sh

```diff
@@ -158,11 +164,17 @@ key_url="https://cli.doppler.com/keys/public"
 if [ "$VERIFY_SIGNATURE" -eq 1 ]; then
   log_debug "Checking for gpg binary"
   if [ ! -x "$(command -v gpg)" ]; then
-    log_debug "Unable to find gpg binary, skipping signature verification"
-    VERIFY_SIGNATURE=0
-    echo "WARNING: Skipping signature verification due to no available gpg binary"
-    echo "Signature verification is an additional measure to ensure you're executing code that Doppler produced"
-    echo "You can remove this warning by installing your system's gnupg package, or by specifying --no-verify-signature"
+    if [ "$FORCE_VERIFY_SIGNATURE" -eq 1 ]; then
+      echo "ERROR: Unable to find gpg binary for signature verficiation"
+      echo "You can resolve this error by installing your system's gnupg package"
+      clean_exit 1
+    else
+      log_debug "Unable to find gpg binary, skipping signature verification"
+      VERIFY_SIGNATURE=0
+      echo "WARNING: Skipping signature verification due to no available gpg binary"
```

Comment:
Just want to make sure that we are okay with printing this. Not sure if this would potentially interfere for user that don't expect any output from our install script (probably not, but just checking).

Reply:
I like your thinking. This script already outputs arbitrary text (though we try to minimize how much) so we should be ok to add this.